ruby/ql/lib/codeql/ruby/frameworks/ActiveStorage.qll

```diff
@@ -85,7 +85,6 @@ module ActiveStorage {
           // Class methods
           API::getTopLevelMember("ActiveStorage")
               .getMember("Blob")
-              .getASubclass()
```

Comment:
Why was this removed?

Reply:
From the PR description:
> The PR also removes some uses of getASubclass() which were redundant because it came before a member predicate that itself performs getASubclass(). The optimizer did not handle that very well.

Reply:
Sorry, I missed that.